code/datums/traits_disused.dm

```diff
@@ -138,6 +138,7 @@
 	points = -1
 	unselectable = 1
 
+/*
```

Comment:
unrelated change?

Reply:
These use the add_memory() proc which has been changed. Code here is unused, so it just needs to be revisited in the future if updating

Reply:
Better to either fix the code or remove it, don't just block comment stuff.

Reply:
there is a proc for dynamic memories used for conspirators, if you want to Do The Needful for this, but considering these are unused, ehhh?